Makefile

```diff
@@ -60,11 +60,9 @@ mk_cmake:
 		-DCMAKE_BUILD_TYPE=$(MK_BUILD_TYPE) \
 		-DCMAKE_INSTALL_PREFIX=$(MK_INSTALL_PATH) \
 		-DKOMPUTE_OPT_INSTALL=1 \
-		-DKOMPUTE_OPT_REPO_SUBMODULE_BUILD=1 \
 		-DKOMPUTE_OPT_BUILD_TESTS=1 \
 		-DKOMPUTE_OPT_BUILD_DOCS=1 \
 		-DKOMPUTE_OPT_BUILD_SHADERS=1 \
```

Comment:
It seems there are still some of the recently depricated flags here

Reply:
Ah, yes. I'm currently not sure, what I should do with the Makefile, since it contains mostly calls specific to your setup? Should I just update it as well, since large parts can be realized now via CMake instead of using a Makefile.

Reply:
Ok I see what you mean, I think the main things that were using my setup was the vcpkg paths, which we should really be in a position where we can basically not use them as the default, so we should make the `makefile` an external component, which means we'll just have to update these commands to match the new flags